packml_smach/src/executeState.py

```diff
@@ -33,7 +37,13 @@ def __init__(self):
         
     def execute(self, userdata):
         rospy.loginfo('Executing state Execute')
-        if userdata.execute_in == True:
+
+        # Receive Stop Tag
+        var4 = config.client.get_node("ns=3;s=\"PackML_Status\".\"UN\".\"Cmd_Stop\"")  
```

Comment:
There's no need to make variable names unique across python files.  Each file has its own scope.